### PR TITLE
Add site-wide Organization JSON-LD from the site config

### PR DESCRIPTION
**Pull request by VPS-julia**
Task: https://vivid-planet.atlassian.net/browse/COM-2967 (parent epic: COM-2963)

## Description

Dextinity exposes a single free-text `structuredData` field per page (in the SEO block). Site-wide entity data like the `Organization` schema has no home, so it would have to be repeated by hand on every page, untyped and prone to drift.

The data is identical on every page and never edited by content editors, so it belongs in the site config rather than in the database. This adds an `organization` section to the site config and renders a typed `Organization` JSON-LD script in the site layout. No entity, no admin UI, no migration.

**Example**

```ts
// demo/site-configs/main.ts
public: {
    organization: {
        name: "Vivid Planet Software GmbH",
        url: "https://www.vivid-planet.com",
        logo: "/assets/dextinity-logo.svg",
        sameAs: ["https://github.com/vivid-planet"],
        description: "Vivid Planet Software GmbH develops Dextinity, the open-source content management system.",
    },
}
```

Only `name` is required. `url` falls back to the site URL, and a relative `logo` path is resolved against it.

### CODE REVIEW

File: demo/api/src/site-settings/entities/site-settings.entity.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    [OptionalProps]?: "createdAt" | "updatedAt";
+
+    @PrimaryKey({ type: "uuid" })
+    @Field(() => ID)
+    id: string = uuid();
+
+    @RootBlock(SiteSettingsContentBlock)
+    @Property({ type: new RootBlockType(SiteSettingsContentBlock) })
+    @Field(() => RootBlockDataScalar(SiteSettingsContentBlock))
+    content: BlockDataInterface;
```
Comment (VPS-thodax) [inline demo/api/src/site-settings/entities/site-settings.entity.ts:23]: What do we gain by making the site settings a block? Intuitively, I would have seen this as a "normal" entity with the fields as DB columns.

I'd also be interested in how @VPS-Obi sees this

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        const input = { content: SiteSettingsContentBlock.state2Output(siteSettingsState) };
+        await update({
+            variables: { input, scope },
+        });
+    };
+
+    return (
+        <Stack topLevelTitle={null}>
```
Comment (VPS-thodax) [inline demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: Do we need a Stack here? There's no StackSwitch and there aren't multiple StackPages, so I don't see why we need it.

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                    <FormattedMessage id="siteSettings.edit.toolbarTitle" defaultMessage="Edit site settings" />
+                </ToolbarTitleItem>
+                <FillSpace />
+                <ToolbarActions>
+                    <SaveButton disabled={!hasChanges} loading={saving} hasErrors={hasSaveErrors != null} onClick={handleSavePage} />
+                </ToolbarActions>
+            </StackToolbar>
+            <MainContent>
+                <BlockAdminComponentRoot>
+                    <SiteSettingsContentBlock.AdminComponent state={siteSettingsState} updateState={setSiteSettingsState} />
```
Comment (VPS-thodax) [inline demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: I'm not a particular fan of the UI (which is probably related to this being a block). I don't like the "Blocks" heading on top. Also I think this would look better if it was a FieldSet

Image

**Comment by VPS-thodax** [comment]
I want to add this also to the starter, so I added the label

File: demo/api/src/site-settings/entities/site-settings.entity.ts (hunk 1 above)
Comment (VPS-julia) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: The ticket states, that there should be a site-settings-content.block.ts  (as a composite block with the OrganizationBlock as a child). 🤔 

Probably it is easier to add additional config blocks in the future? But this can also be done with migrations - so I guess this is not really an argument. I would also prefer not using a block here - also because of the styling issues.

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx (hunk 2 above)
Comment (VPS-julia) [reply demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: You're right, we don't need it here.

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx (hunk 3 above)
Comment (VPS-julia) [reply demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: I tested using a `FieldSet` but I faced some problems:

- The block UI is made for a grey background and in my opinion it looks not very good on white background:

Screenshot 2026-08-04 at 09 28 20


- When using a FieldSet I need a custom implementation for navigating the child blocks, e.g. when adding a URL to `Same as`, which would also need custom styling.

My suggestion is to use `BlockAdminComponentRoot` (as I did before) but set the title correctly to `Organization`, so navigation and styling comes out of the box. 
Screenshot 2026-08-04 at 09 33 27

File: demo/api/src/site-settings/entities/site-settings.entity.ts (hunk 1 above)
Comment (nsams) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: I would not use blocks for this feature, this seems more like structured data and the implemenation would be simpler and shorter without blocks.

The logo still might be a block (because it's a file. As a single DamFileBlock); however, as long as the logo in the header is hardcoded, I don't see why it must be configurable for this Organization. Maybe hardcode it (like in Header) for now and make them configurable (with a single source!) in a follow up PR.

Comment (VPS-Obi) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: I wouldn't make this a block either. However, we could then ask why the footer is a block, since it follows the same schema (entity + content block): https://github.com/vivid-planet/comet/blob/a4b10f6500a3c7208b916cda82e580149485b40f/demo/api/src/footer/entities/footer.entity.ts#L20-L23

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx (hunk 3 above)
Comment (VPS-Obi) [reply demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: IMO it looks worse when using a field set, but I'm fine with both UIs. Either way, we should limit the input field widths. Maybe by using a horizontal form layout (see https://github.com/vivid-planet/comet/pull/1755)?

**Comment by VPS-Obi** [comment]
> I want to add this also to the starter, so I added the label

We should consider if we can move parts to the library if this is to be used in every project. But this can be done in follow-up PRs.

File: demo/api/src/site-settings/blocks/organization.block.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    createBlock,
+    createListBlock,
+    DamImageBlock,
+    ExtractBlockInput,
+} from "@comet/cms-api";
+import { IsNotEmpty, IsString, IsUrl, ValidateIf } from "class-validator";
+
+import { SameAsUrlBlock } from "./same-as-url.block";
+
+const SameAsListBlock = createListBlock({ block: SameAsUrlBlock }, "SameAsList");
```
Comment (VPS-Obi) [inline demo/api/src/site-settings/blocks/organization.block.ts]: Naming: SameAs**Url**ListBlock

File: demo/site/src/app/[visibility]/[domain]/[language]/layout.tsx [hunk 5]
```diff
@@ -61,8 +68,22 @@ export default async function Layout({ children, params }: LayoutProps<"/[visibi
...
+            graphQLFetch,
+            fetch,
+            scope: { domain, language },
+        });
+        organization = buildOrganization(siteSettings.organization, siteConfig.url);
+    }
+
     return (
         <IntlProvider locale={language} messages={messages}>
+            {organization && <JsonLd<Organization> data={organization} />}
```
Comment (VPS-Obi) [inline demo/site/src/app/[visibility]/[domain]/[language]/layout.tsx]: I'd create a `<SiteSettings>` component that builds the organization and renders the structured data. The layout should only fetch the necessary data via the fragment.

File: demo/admin/src/site-settings/blocks/OrganizationBlock.tsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
+import { createCompositeBlock, createCompositeBlockTextField, createListBlock, DamImageBlock } from "@comet/cms-admin";
+import { FormattedMessage } from "react-intl";
+
+import { SameAsUrlBlock } from "./SameAsUrlBlock";
+
+const validateUrl = (value?: string) =>
+    value && !URL.canParse(value) ? <FormattedMessage id="siteSettings.blocks.invalidUrl" defaultMessage="Invalid URL" /> : undefined;
```
Comment (VPS-Obi) [inline demo/admin/src/site-settings/blocks/OrganizationBlock.tsx]: I believe we already have a URL validator in the library.

File: demo/admin/src/siteSettings/blocks/OrganizationBlock.tsx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        },
+        url: {
+            block: createCompositeBlockTextField({
+                label: <FormattedMessage id="siteSettings.blocks.organization.url" defaultMessage="URL" />,
+                helperText: (
+                    <FormattedMessage
+                        id="siteSettings.blocks.organization.url.helperText"
+                        defaultMessage="If empty, the site URL configured for this scope (siteConfig.url) is used."
+                    />
+                ),
```
Comment (VPS-Obi) [inline demo/admin/src/siteSettings/blocks/OrganizationBlock.tsx]: This helper text is too technical, especially the site config part.

File: demo/api/src/site-settings/entities/site-settings.entity.ts (hunk 1 above)
Comment (VPS-Obi) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: Having an OrganizationBlock (as done in https://github.com/vivid-planet/comet/pull/6038/changes/edbb4a4a30eca9c8baa1cc83bb175caa6cae0283) in the entity is okay though. What would be a better alternative? An Organization embeddable?

Comment (nsams) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: > However, we could then ask why the footer is a block, since it follows the same schema (entity + content block):

The footer block is plain content, a block fits pefectly there.

> Having an OrganizationBlock (as done in https://github.com/vivid-planet/comet/commit/edbb4a4a30eca9c8baa1cc83bb175caa6cae0283) in the entity is okay though. What would be a better alternative? An Organization embeddable?

Does it really have to be in an organisation? See Logo for example: that could get used in Header PLUS JSON-LD. Same could be for other fields?

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx (hunk 3 above)
Comment (VPS-julia) [reply demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: It is not possible to use a horizontal layout on the logo and the list 🤔
I don't think there is a way to limit the widths without breaking conventions at the moment...

File: demo/site/src/app/[visibility]/[domain]/[language]/layout.tsx [hunk 8]
```diff
@@ -68,22 +67,9 @@ export default async function Layout({ children, params }: LayoutProps<"/[visibi
...
-            fetch,
-            scope: { domain, language },
-        });
-        organization = buildOrganization(siteSettings.content, siteConfig.url);
-    }
-
     return (
         <IntlProvider locale={language} messages={messages}>
-            {organization && <JsonLd<Organization> data={organization} />}
+            {siteSettings && <SiteSettings siteSettings={siteSettings} scope={{ domain, language }} />}
```
Comment (VPS-Obi) [inline demo/site/src/app/[visibility]/[domain]/[language]/layout.tsx]: IMO it's time to create a `scope` variable before `setNotFoundContext({ domain, language })` and use it throughout 😁

File: demo/admin/src/site-settings/blocks/SameAsUrlBlock.tsx [hunk 9]
```diff
@@ -1,8 +1,8 @@
-import { createCompositeBlock, createCompositeBlockTextField } from "@comet/cms-admin";
+import { createCompositeBlock, createCompositeBlockTextField, isValidUrl } from "@comet/cms-admin";
 import { FormattedMessage } from "react-intl";
 
 const validateUrl = (value?: string) =>
-    value && !URL.canParse(value) ? <FormattedMessage id="siteSettings.blocks.invalidUrl" defaultMessage="Invalid URL" /> : undefined;
+    value && !isValidUrl(value) ? <FormattedMessage id="siteSettings.blocks.invalidUrl" defaultMessage="Invalid URL" /> : undefined;
```
Comment (VPS-Obi) [inline demo/admin/src/site-settings/blocks/SameAsUrlBlock.tsx]: We could export validateUrl from `@comet/cms-admin` and use it here: https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/validation/validateUrl.tsx

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+
+        const input = { organization: OrganizationBlock.state2Output(organizationState) };
+        await update({
+            variables: { input, scope },
+        });
+    };
+
+    return (
+        <>
+            <StackToolbar scopeIndicator={<ContentScopeIndicator />}>
```
Comment (VPS-thodax) [inline demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: Minor detail: This should be a `Toolbar` not a `StackToolbar` since we are not inside a Stack

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx (hunk 3 above)
Comment (VPS-thodax) [reply demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: It seems fine now

File: demo/site/src/siteSettings/buildOrganization.ts [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+function buildLogoUrl(logo: DamImageBlockData, siteUrl: string): string | undefined {
+    const props = logo.block?.props;
+
+    if (!props) {
+        return undefined;
+    }
+
+    if ("urlTemplate" in props && props.damFile?.image) {
+        const { width, height } = props.damFile.image;
+        const url = generateImageUrl({ src: props.urlTemplate, width }, width / height);
```
Comment (VPS-thodax) [inline demo/site/src/siteSettings/buildOrganization.ts]: For me the image URL didn't work for PixelImages (for SVGs it worked fine). I got a validation error:

```
{
  "statusCode": 400,
  "message": "Validation failed",
  "error": "CometValidationException",
  "validationErrors": [
    {
      "target": {
        "hash": "6013e9d8746f0150f034a13ed9f089688628733c",
        "contentHash": "c210c292e35d1fc51c6490e75fe0b1ba",
        "fileId": "de1976b2-41db-4d5c-8bc8-08bc1776130c",
        "focalPoint": "SMART",
        "resizeWidth": 1024,
        "resizeHeight": 1024,
        "filename": "astronaut"
      },
      "value": 1024,
      "property": "resizeWidth",
      "children": [],
      "constraints": {
        "IsAllowedImageSize": "resizeWidth is not allowed"
      }
    }
  ]
}
```

because the width is not in the allowedWidths.

I think we need something similar to here to ensure only valid sizes are used: https://github.com/vivid-planet/comet/blob/ba777cfafcc8c215e9dd3695d62fe48e89a1de91/packages/mail-react/src/blocks/pixelImage/usePixelImageBlockData.ts#L53

File: demo/site/src/siteSettings/buildOrganization.ts [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+    return undefined;
+}
+
+export function buildOrganization(organization: OrganizationBlockData, siteUrl: string): WithContext<Organization> | null {
+    const name = organization.name.trim();
+
+    if (!name) {
+        return null;
+    }
```
Comment (VPS-thodax) [inline demo/site/src/siteSettings/buildOrganization.ts]: It's currently possible to set multiple spaces as name in the admin. This then causes the site to not render any organization info at all because here the name is trimmed.

The trim should happen in the admin before submitting. Or the validation in the admin and api needs to be stricter

File: demo/api/src/site-settings/blocks/same-as-url.block.ts [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+import { IsUrl } from "class-validator";
+
+class SameAsUrlBlockData extends BlockData {
+    @BlockField()
+    url: string;
+}
+
+class SameAsUrlBlockInput extends BlockInput {
+    @BlockField()
+    @IsUrl()
```
Comment (VPS-thodax) [inline demo/api/src/site-settings/blocks/same-as-url.block.ts]: An empty `sameAs` row makes the whole form unsaveable in the admin. There also isn't a good error.

This is because of the IsUrl validation here. It should probably run conditionally with ValidateIf (like in organization.block.ts).

Comment (VPS-thodax) [inline demo/api/src/site-settings/blocks/same-as-url.block.ts]: Minor: The admin validation passes for urls with surrounding whitespaces (e.g., `"  https://example.com  "`) because new URL strips whitespaces. But the api validation fails because IsUrl doesn't strip it. 

You could use a Transform like this:

```ts
@BlockField()
@Transform(({ value }) => (typeof value === "string" ? value.trim() : value))
@ValidateIf((input: SameAsUrlBlockInput) => Boolean(input.url))
@IsUrl({ protocols: ["http", "https"], require_protocol: true })
url: string;
```

or make the admin validation stricter

File: demo/admin/src/site-settings/blocks/SameAsUrlBlock.tsx [hunk 14]
```diff
@@ -0,0 +1,16 @@
+import { createCompositeBlock, createCompositeBlockTextField, validateUrl } from "@comet/cms-admin";
+import { FormattedMessage } from "react-intl";
+
+export const SameAsUrlBlock = createCompositeBlock({
+    name: "SameAsUrl",
+    displayName: <FormattedMessage id="siteSettings.blocks.sameAsUrl.displayName" defaultMessage="URL" />,
+    blocks: {
+        url: {
+            block: createCompositeBlockTextField({
```
Comment (VPS-thodax) [inline demo/admin/src/site-settings/blocks/SameAsUrlBlock.tsx]: The preview for these URLs only shows "URL":

Image

So I have to click on them to see the actual URL. 

Please add a `previewContent` showing the actual url

File: demo/api/src/site-settings/blocks/organization.block.ts [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+class OrganizationBlockInput extends BlockInput {
+    @BlockField()
+    @IsString()
+    @IsNotEmpty()
+    name: string;
+
+    @BlockField()
+    @ValidateIf((input: OrganizationBlockInput) => Boolean(input.url))
+    @IsUrl()
```
Comment (VPS-thodax) [inline demo/api/src/site-settings/blocks/organization.block.ts]: Validation in the API is looser than in admin. Admin requires http or https protocol. You could make them more similar:


```suggestion
    @IsUrl({ protocols: ["http", "https"], require_protocol: true })
```

(same for the sameAs url field)

File: demo/admin/src/siteSettings/blocks/OrganizationBlock.tsx
Comment (VPS-thodax) [inline demo/admin/src/siteSettings/blocks/OrganizationBlock.tsx:1]: Minor: In admin we usually use camelCase for folder names. Here it's kebab-case. So it should be `/siteSettings/` instead of `/site-settings`.

File: demo/admin/src/common/MasterMenu.tsx [hunk 16]
```diff
@@ -192,6 +193,15 @@ const getMasterMenuData = ({ brevoContactConfig }: { brevoContactConfig: BrevoCo
                     },
                     requiredPermission: "pageTree",
                 },
+                {
+                    type: "route",
+                    primary: <FormattedMessage id="menu.project-snips.siteSettings" defaultMessage="Site Settings" />,
```
Comment (VPS-thodax) [inline demo/admin/src/common/MasterMenu.tsx]: Minor: We usually use sentence case, so it should be:


```suggestion
                    primary: <FormattedMessage id="menu.project-snips.siteSettings" defaultMessage="Site settings" />,
```

(though this isn't done correctly everywhere in the Menu 😅)

File: demo/api/src/site-settings/entities/site-settings.entity.ts (hunk 1 above)
Comment (VPS-thodax) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: > Does it really have to be in an organisation? See Logo for example: that could get used in Header PLUS JSON-LD. Same could be for other fields?

True, we could think the whole "site settings" bigger. But that makes it more complicated and puts us at risk of a) adding unnecessary complexity and b) getting stuck at this point because we can't decide how to move forward.

Since this isn't a library feature but a pure in-project implementation, I'd suggest to leave it like it is. 
If a project has other requirements, they can change it up completely. And we can also change it again in the future without introducing a breaking change. (of course, projects might be stuck with the old version then, but that's an acceptable risk IMO)

So for me it's okay to leave it like this

**Review by VPS-thodax — COMMENTED** [review]
As a tip: I did some manual testing but I also let Claude test this feature with playwright and it's way better at identifying and testing edge cases. I strongly recommend using it for testing a feature too.

File: demo/api/src/site-settings/blocks/organization.block.ts (hunk 15 above)
Comment (VPS-julia) [reply demo/api/src/site-settings/blocks/organization.block.ts]: In admin I use `validateUrl` imported from cms-admin, which does not check for the protocol: https://github.com/vivid-planet/comet/blob/main/packages/admin/cms-admin/src/validation/validateUrl.tsx

Do you think, it is necessary to use stricter validation? (by adding a new validator to the cms-admin package or create one in the demo...?)

File: demo/admin/src/siteSettings/blocks/OrganizationBlock.tsx
Comment (VPS-julia) [reply demo/admin/src/siteSettings/blocks/OrganizationBlock.tsx:1]: Fixed in 398d0f70c

File: demo/admin/src/common/MasterMenu.tsx (hunk 16 above)
Comment (VPS-julia) [reply demo/admin/src/common/MasterMenu.tsx]: Fixed in e6b353c58

File: demo/admin/src/site-settings/EditSiteSettingsPage.tsx (hunk 10 above)
Comment (VPS-julia) [reply demo/admin/src/site-settings/EditSiteSettingsPage.tsx]: Fixed in f683748bf

File: demo/admin/src/site-settings/blocks/SameAsUrlBlock.tsx (hunk 14 above)
Comment (VPS-julia) [reply demo/admin/src/site-settings/blocks/SameAsUrlBlock.tsx]: Fixed in 71293b245

File: demo/site/src/siteSettings/buildOrganization.ts (hunk 12 above)
Comment (VPS-julia) [reply demo/site/src/siteSettings/buildOrganization.ts]: Fixed in d2c8bfc20

File: demo/api/src/site-settings/blocks/same-as-url.block.ts (hunk 13 above)
Comment (VPS-julia) [reply demo/api/src/site-settings/blocks/same-as-url.block.ts]: Fixed in 09ceb2bea

Comment (VPS-julia) [reply demo/api/src/site-settings/blocks/same-as-url.block.ts]: Fixed in b930b4b32

File: demo/site/src/siteSettings/buildOrganization.ts (hunk 11 above)
Comment (VPS-julia) [reply demo/site/src/siteSettings/buildOrganization.ts]: Fixed in 5cd9e0a93

File: demo/api/src/site-settings/blocks/organization.block.ts (hunk 15 above)
Comment (VPS-Obi) [reply demo/api/src/site-settings/blocks/organization.block.ts]: Currently Admin and API use isUrl from class-validator, so the validation should be equally strict. If we want to check the protocol here, we shouldn't use the validateUrl helper from cms-admin. Changing the helper in the library would be a breaking change.

**Review by VPS-Obi — DISMISSED** [review]

File: demo/api/src/site-settings/blocks/organization.block.ts (hunk 15 above)
Comment (VPS-thodax) [reply demo/api/src/site-settings/blocks/organization.block.ts]: I think we need the protocol check because the structured data is only valid if the URL has a protocol. Unless we parse the URL in the site and always add a https protocol there

File: .changeset/tidy-pears-attack.md [hunk 17]
```diff
@@ -0,0 +1,14 @@
+---
+"@comet/cms-admin": minor
```
Comment (VPS-thodax) [inline .changeset/tidy-pears-attack.md]: Please rebase on main and do the dextinity renaming steps

File: demo/api/src/site-settings/entities/site-settings.entity.ts (hunk 1 above)
Comment (nsams) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: If it is not a bigger site settings feature then it should also not be named like that.

If it is the Organisation only: I'm sceptic about adding 800 LOC to every new project just for this Json-LD that is the same on every page. I highly doubt it has any SEO/GEO effect. (I'm not talking about Products/Articels/News etc Json-LD here)

File: demo/api/src/site-settings/blocks/organization.block.ts (hunk 15 above)
Comment (VPS-julia) [reply demo/api/src/site-settings/blocks/organization.block.ts]: I reverted the export of validateUrl and introduced another helper including a check for the protocol in demo: [f0c6e58](https://github.com/vivid-planet/dextinity/pull/6038/commits/f0c6e58812f6bb24b241da1e0283f4fe27243f55)

**Review by VPS-thodax — DISMISSED** [review]

File: demo/api/src/site-settings/entities/site-settings.entity.ts (hunk 1 above)
Comment (VPS-julia) [reply demo/api/src/site-settings/entities/site-settings.entity.ts:23]: @thomasdax98 @nsams @VPS-Obi 
I removed the api/admin implementation. The data for the Json-LD comes from the siteConfig now.  I also updated the PR description.

File: demo/site-configs/main.ts [hunk 18]
```diff
@@ -20,6 +20,13 @@ export default ((env) => {
                 languages: ["en", "de"],
             },
             recaptchaSiteKey: process.env.RECAPTCHA_SITE_KEY ?? "",
+            organization: {
+                name: "Vivid Planet Software GmbH",
+                url: "https://www.vivid-planet.com",
+                logo: "/assets/dextinity-logo.svg",
+                sameAs: ["https://github.com/vivid-planet"],
+                description: "Vivid Planet Software GmbH develops Dextinity, the open-source content management system.",
```
Comment (VPS-Obi) [inline demo/site-configs/main.ts]:
```suggestion
                description: "Vivid Planet Software GmbH develops Dextinity CMS, the open-source content management system.",
```

**Review by VPS-Obi — DISMISSED** [review]

**Review by VPS-Obi — APPROVED** [review]

**Review by nsams — APPROVED** [review]

**Review by VPS-thodax — APPROVED** [review]
